src/cirq_iqm/iqm_sampler.py

```diff
@@ -23,12 +23,14 @@
 import numpy as np
 from cirq import study
 from cirq.study import resolver
+from iqm_client import iqm_client
+from iqm_client.iqm_client import IQMClient, SingleQubitMapping
```

Comment:
It would be more consistent to either import all classes directly, or use `iqm_client.ClassXXX` for all of them.

Reply:
The latter seems to be the recommended way, more informative to the reader.

Reply:
This was discussed and I switched to this style because having a `Circuit` (as opposed to `iqm_client.Circuit`) was deemed confusing (because there's also `cirq.Circuit`). I'm personally fine with using just `Circuit` or `iqm_client.Circuit` but I wouldn't want to preface all imports with `iqm_client` when there is no risk of confusion with Cirq types.